src/app/store/reducers/login.reducers.ts

```diff
@@ -0,0 +1,26 @@
+import { createReducer, on } from "@ngrx/store";
+import { initialLoginState } from "../state/login.state";
+import { LoginUser, LoginUserSuccess, LoginUserError } from "../actions/login.actions";
+
+
+export const loginReducers = createReducer(
+    initialLoginState,
+
+  on(LoginUser, (state) => ({
+    ...state,
+    loading: true,
+  })),
+
+  on(LoginUserSuccess, (state, { user }) => ({
+    ...state,
+    currentUser: user,
+    status: "success",
+    error: "",
```

Comment:
I see no reason really to keep the error on the state. Just leave it in the payload of the error action and show it in the effect.